spec/workspace-spec.coffee

```diff
@@ -777,7 +777,7 @@ describe "Workspace", ->
         })
         workspace2.deserialize(atom.workspace.serialize(), atom.deserializers)
         item = atom.workspace.getActivePaneItem()
-        expect(document.title).toBe "#{item.getTitle()} - #{atom.project.getPaths()[0]} - Atom"
+        expect(document.title).toMatch ///^#{item.getTitle()}\ \u2014\ #{atom.project.getPaths()[0]}///
```

Comment:
The end of the title is variable based on platform. These new tests only cover the platform-agnostic portions, from what I saw anyway.